javascript-light-bulb/main.js

```diff
@@ -0,0 +1,18 @@
+let numberOfClicks = 0;
+
+const $lightBulb = document.querySelector('.light-bulb');
+const $row = document.querySelector('.row');
+
+$lightBulb.addEventListener('click', handleClick);
+
+function handleClick(event) {
+  numberOfClicks++;
+  console.log(numberOfClicks);
```

Comment:
Remove all `console.log`s before submitting. They are only for testing / debugging purposes. Unless we specifically ask for them in the instructions

Reply:
![Screenshot 2023-07-25 at 11 54 25 AM](https://github.com/aaronsoukaphay/c0723-code-solutions/assets/132744086/c038dd70-1d5b-4075-aaeb-ed0109a3eb78)